code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -595,8 +596,6 @@ Difficulty: Hard
 		var/limb_to_hit = L.get_bodypart(pick("head", "chest", "r_arm", "l_arm", "r_leg", "l_leg"))
 		var/armor = L.run_armor_check(limb_to_hit, "melee", "Your armor absorbs [src]!", "Your armor blocks part of [src]!", 50, "Your armor was penetrated by [src]!")
 		L.apply_damage(damage, BURN, limb_to_hit, armor)
-		if(ismegafauna(L) || istype(L, /mob/living/simple_animal/hostile/asteroid))
-			L.adjustBruteLoss(damage)
```

Comment:
And on station, spamming forever doesn't really matter if the weapon itself is weak, since if someone with a burst weapon kills you first your ammo doesn't do anything.